tabs/src/Wallet.tsx

```diff
@@ -1,22 +1,21 @@
-import React from "react";
-import WalletAllowanceCard from "./components/WalletAllowanceCard";
-import WalletTransactionHistory from "./components/WalletTransactionHistory";
-import WalletTransactionLog from "./components/WalletTransactionLog";
+import React from 'react';
+import WalletAllwanaceMainComponent from './components/WalletAllwanaceMainComponent';
 
 const Wallet: React.FC = () => {
-
-
   return (
     <div>
       <div>
-        <WalletAllowanceCard
+        {/* <WalletAllowanceCard
           availableSats={2000000}
           availableAmountUSD={74.2}
           remainingSats={90000}
           spentSats={20000}
         />
         <WalletTransactionHistory />
         <WalletTransactionLog />
+         */}
+
+        <WalletAllwanaceMainComponent />
```

Comment:
Suggest we pass in the wallet ID as a property into the Component. E.g. `<WalletAllowanceComponent WalletId="1234567890"/>`